### PR TITLE
website: Update emanote

### PR DESCRIPTION
**Pull request by srid**
- PrismJs is updated, so syntax highlighting on the website is more reliable
- MathJax rendering enabled on website

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ To discuss the projects and the content under the Plutonomicon umbrella, join ou
 
 If you'd like to run a live version of the website locally:
 
-```bash
+```sh-session
```
Comment (L-as) [inline README.md:53]: What is all this?

File: flake.lock [hunk 2]
```diff
@@ -232,9 +294,9 @@
     },
     "hercules-ci-effects": {
       "inputs": {
-        "flake-compat": "flake-compat_4",
+        "flake-compat": "flake-compat_6",
         "hercules-ci-agent": "hercules-ci-agent",
-        "nixpkgs": "nixpkgs_4",
+        "nixpkgs": "nixpkgs_6",
```
Comment (L-as) [inline flake.lock]: Why do we have so many nixpkgses. We probably need some follows.

File: README.md (hunk 1 above)
Comment (srid) [reply README.md:53]: That's language identifier for shell-session. cf. https://github.com/PrismJS/prism/pull/1892

File: flake.lock (hunk 2 above)
Comment (srid) [reply flake.lock]: You'd have to add follows _recursively_ to every flake input.

Comment (L-as) [reply flake.lock]: What do you mean? If N dependencies each use 1 Nixpkgs, we should only need at most N follows to only use 1 Nixpkgs.

Comment (srid) [reply flake.lock]: You have to do that for dependencies of dependencies too.

```
...
emanote -> tailwind-haskell -> flake-utils -> nixpkgs
emanote -> ema -> flake-utils -> nixpkgs
emanote -> ema -> flake-compat
...
```

All those paths should be set, which I think I did in recent commits.

See https://github.com/srid/emanote/blob/bd2526efc3af2d88590b608c31f89aa115a8840e/flake.nix#L7-L12

Comment (L-as) [reply flake.lock]: That is in fact true. This is O(n^2) for n dependencies it seems.

**Review by L-as — APPROVED** [review]
LGTM

**Pull request merged**